lib/src/models/view/timer.dart

```diff
@@ -0,0 +1,37 @@
+import "package:flutter/foundation.dart";
+
+/// Contains the duration of a current mission
+class MissionTimer with ChangeNotifier{
+  /// Name of the mission 
+  final String name;
+
+  /// The current duration of a mission
+  Duration duration;
+  
+  /// The time which the timer should end
+  late DateTime end;
```

Comment:
```suggestion
  /// The time which the timer should end
  final DateTime end;
```
1. You can get rid of `duration` since you never use it
2. You can make `end` final, see the changes to the constructor below